### PR TITLE
Encode stream URLs to shield them off from Plex

### PR DESCRIPTION
**Pull request by HazCod**
Hi @tombowditch , this is what we had in mind.
To be tested with real M3Us.

Fixes https://github.com/tombowditch/telly/issues/77

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -124,10 +125,14 @@ func buildChannels(usedTracks []m3u.Track) []LineupItem {
 			finalName = track.TvgName
 		}
 
+		// base64 url
+		trackUri := base64.StdEncoding.EncodeToString([]byte(track.URI))
+		fullTrackUri := fmt.Sprintf("http://%s", *listenAddress) + "/stream/" + trackUri
```
Comment (tombowditch) [inline main.go]: What if my `listenAddress` is 0.0.0.0?

Comment (HazCod) [reply main.go]: Good point!

**Comment by HazCod** [comment]
Confirmed working as in the issue

**Comment by tombowditch** [comment]
This is looking very good - is there any underlying issues? I should test it shortly.

**Comment by tombowditch** [comment]
Looks good!

**Comment by tombowditch** [comment]
Releasing v0.5 shortly

**Pull request merged**